Source/Lib/Codec/EbEncHandle.c

```diff
@@ -2399,19 +2399,22 @@ static EB_ERRORTYPE VerifySettings(\
         return_error = EB_ErrorBadParameter;
     }
 
-    if ((config->level != 0) && (config->rateControlMode) && (config->tier == 0) && ((config->targetBitRate*2) > mainTierMaxBitRate[levelIdx])){
+    uint32_t maxBitrate = config->vbvMaxrate ? config->vbvMaxrate : config->targetBitRate * 2;
```

Comment:
hate seeing magic numbers here.  Can 2 and 3 be constants with an appropriate name, or perhaps a comment why double or triple the target bitrate is being used.